src/components/templates/Page.tsx

```diff
@@ -30,6 +30,7 @@ export interface PageProps {
   description?: string
   noPageHeader?: boolean
   headerCenter?: boolean
+  isEdgeNetwork?: boolean
```

Comment:
suggestion(blocking): I think `isEdgeNetwork` is a bit confusing. Maybe renaming this to `isEdgeProvider` makes more sense here? (Applies to all occurences)